interfaces/HO-Portal/src/app/program/program-payout/program-payout.component.ts

```diff
@@ -196,6 +203,19 @@ export class ProgramPayoutComponent implements OnChanges {
 
     await alert.present();
   }
+
+  private checkPhaseReady() {
+    // This component only influences ready in the 'payment'-phase
+    if (this.activePhase === ProgramPhase.payment) {
```

Comment:
Often using a 'return as early as possible'-approach makes for a short and readable method.

In this case: 
```
if (this.activePhase !== ProgramPhase.payment) {
  return this.emitCompleted.emit(true);
}
// from this point down there is only 1 'thruth' and no 'else' to worry about/take care of all these dangling `}`'s.
```
This can be done multiple times in every method. It saves you from having multiple levels of indentation and make for a better 'overview' of what happens inside a method.